regions_www/m/server.R

```diff
@@ -22,6 +22,9 @@
 ## Functions
 source("pct_shiny_funs.R", local = T)
 
+## Enable bookmarking at the server side
+enableBookmarking(store = "server")
```

Comment:
That's a fair point. The only big problem is (apart from a messy URL for a specific states), the URL keeps changing when someone uses the app and they are not even interested to save or restore the app's state. On the bright side, it's easier to manage, as the servers don't need to remember anything.
I still believe that server-side bookmarking is much neater. I can think of two solutions here:
1) We don't care about app's state not saved on a specific server
2) For the two servers, we routinely mirror their saved states. This requires writing a routine which runs periodically to manage. 

What do you think?

Reply:
Not sure what you mean with 
> the URL keeps changing when someone uses the app

I've switched to `store = "url"` and in my browser the URL doesn't change when I'm moving around.  Does it in yours?

Does solution (1) mean we provide a bookmark system that works 50% of the time?  Not sure I like that.

Solution (2) is possible but I've not got budgeted time to do it, also I think it is likely to be a hacky fix.  Even per server we have multiple instances of R running which could well interact in surprising ways.

Server side results in a neater URL but a much messier code.

I've pushed https://github.com/npct/pct-shiny/pull/659/commits/fdd151bd05615fa1ca98ba2ed4ff258aa0a62c0a as a WIP but really I think it might be easier with the long messy URL sadly.